scripts/hotWalletCli/constants.ts

```diff
@@ -0,0 +1,7 @@
+import os from 'node:os'
+import path from 'node:path'
+
+export const RFOX_DIR = path.join(os.homedir(), 'rfox')
```

Comment:
Does this assume `~/rfox`?

Reply:
correct, it will create if it doesn't already exist and is a unified location to store all relevant recovery files